src/docs/product/issues/issue-details/performance-issues/regressions.mdx

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
unless we want both statistical detector issues on the same page, could you change the route from `regressions` to `function-regressions`?